modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -0,0 +1,618 @@
+    $sites = array('' => 'All');
+    if ($user->hasPermission('access_all_profiles')) {
+        // get the list of study sites - to be replaced by the Site object
+        $sites = Utility::getSiteList();
+        if (is_array($sites)) {
+            $sites = array('' => 'All') + $sites;
+        }
+    } else {
+        // allow only to view own site data
+        $site =& Site::singleton($user->getData('CenterID'));
+        if ($site->isStudySite()) {
+            $sites[$user->getData('CenterID')] = $user->getData('Site');
+        }
+    }
+
+    //not yet ideal permissions
+    $assignees = array();
+    if ($user->hasPermission('access_all_profiles')) {
+        $assignee_expanded = $db->pselect(
+            "SELECT Real_name, UserID FROM users",
+            array()
+        );
+    } else {
+        $CenterID = $user->getCenterID();
+        $DCCID = $db->pselectOne("SELECT CenterID from psc where Name='DCC'",
+            array());
+        $assignee_expanded = $db->pselect(
+            "SELECT u.Real_name, u.UserID FROM users u 
+WHERE (u.CenterID=:CenterID) OR (u.CenterID=:DCC)",
+            array('CenterID' => $CenterID,
+                'DCC' => $DCCID)
+        );
+    }
+
+    foreach ($assignee_expanded as $a_row) {
+        $assignees[$a_row['UserID']] = $a_row['Real_name'];
+    }
+
+    $otherWatchers = array();
+    $potential_watchers_expanded = $db->pselect(
+        "SELECT Real_name, UserID FROM users",
+        array()
+    );
+    foreach ($potential_watchers_expanded as $w_row) {
+        $otherWatchers[$w_row['UserID']] = $w_row['Real_name'];
+    }
+
+    //can't set to closed if not developer.
+    if ($user->hasPermission('issue_tracker_developer')) {
+        $statuses = array(
+            'new' => 'New',
+            'acknowledged' => 'Acknowledged',
+            'assigned' => 'Assigned',
+            'feedback' => 'Feedback',
+            'resolved' => 'Resolved',
+            'closed' => 'Closed',
+        );
+    } else {
+        $statuses = array(
+            'new' => 'New',
+            'acknowledged' => 'Acknowledged',
+            'assigned' => 'Assigned',
+            'feedback' => 'Feedback',
+            'resolved' => 'Resolved',
+        );
+    }
+
+    $priorities = array(
+        'low' => 'Low',
+        'normal' => 'Normal',
+        'high' => 'High',
+        'urgent' => 'Urgent',
+        'immediate' => 'Immediate',
+    );
+
+    $categories = array(
+        'Behavioural Instruments' => 'Behavioural Instruments',
+        'Behavioural Battery' => 'Behavioural Battery',
+        'Data Entry' => 'Data Entry',
+        'Database Problems' => 'Database Problems',
+        'Examiners' => 'Examiners',
+        'SubprojectID/Project/Plan Changes' => 'SubprojectID' .
+            '/Project/Plan Changes',
+        'Imaging' => 'Imaging'
+    );
+
+    $modules = array();
+    $modules_expanded = $db->pselect(
+        "SELECT DISTINCT Label, ID FROM LorisMenu ORDER BY Label",
```

Comment:
again: And Parent is not null --> to exclude top menus (Candidate, Clinical, Tools, Reports, Admin)
 Add a line note